2.4/root/usr/share/container-scripts/mongodb/replset_petset_supervisor.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Now that are adding a new "supervisor.sh" script, I wonder, could we document on the top of the script what is its purpose?